src/client/dfuse/dfuse_main.c

```diff
@@ -136,6 +136,7 @@ main(int argc, char **argv)
 		{"sys-name",		required_argument, 0, 'G'},
 		{"mountpoint",		required_argument, 0, 'm'},
 		{"singlethread",	no_argument,	   0, 'S'},
+		{"enable-caching",	no_argument,	   0, 'A'},
```

Comment:
A for caching?

Reply:
The A isn't used, it's just you need a single character to be able to identify the long option.